src/types/contracts/registrar.ts

```diff
@@ -47,20 +68,23 @@ export type VaultListRes = {
 };
 
 export type RegistrarConfigPayload = {
-  accounts_code_id?: number;
-  index_fund_contract?: string; //addr
-  treasury?: string; //addr
-  tax_rate?: string; //decimal string
-  // approved_charities?: string[];
-  default_vault?: string;
-  guardians_multisig_addr?: string;
-  endowment_owners_group_addr?: string;
+  accounts_contract?: string;
+  index_fund_contract?: string;
+  treasury?: string;
   split_max?: string; //decimal string
   split_min?: string; //decimal string
   split_default?: string; //decimal string
   halo_token?: string;
   gov_contract?: string;
   charity_shares_contract?: string;
+  rebalance?: RebalanceDetails;
+  cw3_code?: number;
+  cw4_code?: number;
+  accepted_tokens_native?: string[];
```

Comment:
```suggestion
  //accepted_tokens_native?: string[];
```
nit: temp comment out fields not used in `<Form/>`
to avoid such fields being used as fieldnames
e.g
```tsx
<Field<RV>
          label="Impact Applications Review"
          name="rebalance" //<--- valid
        /> 
```

Reply:
Rebalance items have been removed in [8fb445c](https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/1847/commits/8fb445c1a0df6355064c627143602217c061f252).